content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -11,7 +11,7 @@ weight: 10
 
 <!-- overview -->
 
-Kubernetes {{< skew currentVersion >}} supports [Container Network Interface](https://github.com/containernetworking/cni)
+Kubernetes since earliest releases, including {{< skew currentVersion >}} supports [Container Network Interface](https://github.com/containernetworking/cni)
```

Comment:
I'm unsure that this is a meaningful improvement to the documentation in terms of usefulness to the reader. If we are going to clarify here, we should include the specific version in which CNI plugin support was first introduced.

Reply:
We could do a related change:
```diff
-You must use a CNI plugin that is compatible with the 
-[v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md) or later
-releases of the CNI specification. The Kubernetes project recommends using a plugin that is
+For Kubernetes {{< skew currentVersion >}} specifically, you must use a CNI plugin
+that is compatible with the
+[v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md) or later
+releases of the CNI specification.
+The Kubernetes project further recommends using a plugin that is
 compatible with the [v1.0.0](https://github.com/containernetworking/cni/blob/spec-v1.0.0/SPEC.md)
 CNI specification (plugins can be compatible with multiple spec versions).
+If you are running a version of Kubernetes other than v{{< skew currentVersion >}}, check the
+documentation for the release you're running.
```

Reply:
@katcosgrove , my motivation with this change was to indicate that it is not just the latest, but CNI has been standard since kubernetes 1.3.0. Prior to that it was kube-net. So, want to indicate that.

Reply:
I am open to rewording it to indicate this information.

Reply:
```suggestion
Kubernetes (version 1.3 through to the latest {{< skew latestVersion >}}, and likely onwards) lets you use
[Container Network Interface](https://github.com/containernetworking/cni)
```